lib/Model.js

```diff
@@ -38,47 +38,63 @@ Model.ref = jsong.ref;
+ * where the cache size is calculated in terms of graph node count and, for arrays and strings, element count.
+ * @property {number} [collectRatio] The ratio of the maximum size to collect when the maxSize is exceeded.
+ * @property {number} [maxRetries] The maximum number of times that the Model will attempt to retrieve the value from
+ * its DataSource. Defaults to `3`.
+ * @property {Model~errorSelector} [errorSelector] A function used to translate errors before they are returned
+ * @property {Model~onChange} [onChange] A function called whenever the Model's cache is changed
+ * @property {Model~comparator} [comparator] A function called whenever a value in the Model's cache is about to be
+ * replaced with a new value.
+ * @property {boolean} [disablePathCollapse] Disables the algorithm that collapses paths on GET requests. The algorithm
+ * is enabled by default. This is a relatively computationally expensive feature.
+ * @property {boolean} [disableRequestDeduplication] Disables the algorithm that deduplicates paths across in-flight GET
+ * requests. The algorithm is enabled by default. This is a computationally expensive feature.
+ */
+
 /**
  * A Model object is used to execute commands against a {@link JSONGraph} object. {@link Model}s can work with a local JSONGraph cache, or it can work with a remote {@link JSONGraph} object through a {@link DataSource}.
  * @constructor
- * @param {?Object} options - a set of options to customize behavior
- * @param {?DataSource} options.source - a data source to retrieve and manage the {@link JSONGraph}
- * @param {?JSONGraph} options.cache - initial state of the {@link JSONGraph}
- * @param {?number} options.maxSize - the maximum size of the cache. This value roughly correlates to item count (where itemCount = maxSize / 50). Each item by default is given a metadata `$size` of 50 (or its length when it's an array or string). You can get better control of falcor's memory usage by tweaking `$size`
- * @param {?number} options.maxRetries - the maximum number of times that the Model will attempt to retrieve the value from the server.
- * @param {?number} options.collectRatio - the ratio of the maximum size to collect when the maxSize is exceeded
- * @param {?Model~errorSelector} options.errorSelector - a function used to translate errors before they are returned
- * @param {?Model~onChange} options.onChange - a function called whenever the Model's cache is changed
- * @param {?Model~comparator} options.comparator - a function called whenever a value in the Model's cache is about to be replaced with a new value.
+ * @param {Options} [o] - a set of options to customize behavior
  */
 function Model(o) {
-
     var options = o || {};
     this._root = options._root || new ModelRoot(options);
     this._path = options.path || options._path || [];
-    this._scheduler = options.scheduler || options._scheduler || new ImmediateScheduler();
```

Comment:
love removing this property given that it is indeed never used outside of initializing `_request` below, especially because there are many models derived from the root one, but almost never a new RequestQueue